lib/types/zpool.nix

```diff
@@ -63,19 +63,79 @@ in
                         https://openzfs.github.io/openzfs-docs/man/master/7/zpoolconcepts.7.html#Virtual_Devices_(vdevs)
                         for details.
                       '';
-                      example = [{
-                        mode = "mirror";
-                        members = [ "x" "y" "/dev/sda1" ];
```

Comment:
I also adjusted this example to

1. be in line with the others
2. actually be valid (passing in a path like `/dev/sda` results in the script failing because `all_devices_list` and `zfs_devices_list` won't match).